ShopifySharp/Services/FulfillmentOrders/FulfillmentService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net.Http;
+using ShopifySharp.Filters;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ShopifySharp.Infrastructure;
+using System;
+using System.Threading;
+using ShopifySharp.Lists;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// A service for manipulating Shopify fulfillment orders.
+    /// </summary>
+    public class FulfillmentOrderService : ShopifyService
+    {
+        /// <remarks>
+        /// This endpoint is not supported in 2019-10.
+        /// </remarks>
+        public override string APIVersion => "2020-07";
```

Comment:
We can remove this override since another PR was merged that bumps the targeted API version to `2020-10`.